package.json

```diff
@@ -41,6 +41,7 @@
     "babel-jest": "^23.6.0",
     "core-js": "^2.6.10",
     "coveralls": "^3.0.7",
+    "dompurify": "^2.0.6",
```

Comment:
I am not sure we need this one, if we don't sanitize. Probably we don't need to do it in story as well.

Reply:
It's more about dangerous data (script tags, any malicious code, etc.). No, I agree, it's a good practice to treat any source as not trusted. :+1: